jctools-core/src/main/java/org/jctools/queues/BaseMpscLinkedArrayQueue.java

```diff
@@ -73,13 +77,22 @@ final boolean casProducerIndex(long expect, long newValue)
     private volatile long consumerIndex;
     protected long consumerMask;
     protected E[] consumerBuffer;
+    private volatile E[] volatileConsumerBuffer;
```

Comment:
why do we need 2 consumer buffers?

Reply:
Should I make consumerBuffer volatile and add lp/lv/svConsumerBuffer methods instead?

Reply:
why does it need to be volatile?

Reply:
I wasn't aware that non-volatile reference writes/reads were always atomic. I guess it doesn't need to be volatile. Thanks.